tshirts.cpp

```diff
@@ -1,22 +1,57 @@
 #include <iostream>
 #include <assert.h>
 
+#define UPPER_BOUNDARY_FOR_SMALL   (38)
+#define LOWER_BOUNDARY_FOR_MEDIUM  (39)
+#define UPPER_BOUNDARY_FOR_MEDIUM  (41)
+#define LOWER_BOUNDARY_FOR_LARGE   (42)
+#define SIZE_ZERO                  (0)
+
+void testForSizeZero();
+void testForSmall();
+void testForMedium();
+void testForLarge();
+
 char size(int cms) {
     char sizeName = '\0';
-    if(cms < 38) {
+    if((cms != SIZE_ZERO)&&(cms <= UPPER_BOUNDARY_FOR_SMALL)) {
         sizeName = 'S';
-    } else if(cms > 38 && cms < 42) {
+    } else if((cms > UPPER_BOUNDARY_FOR_SMALL) && (cms < LOWER_BOUNDARY_FOR_LARGE)) {
         sizeName = 'M';
-    } else if(cms > 42) {
+    } else if(cms >= LOWER_BOUNDARY_FOR_LARGE) {
         sizeName = 'L';
     }
     return sizeName;
 }
 
+void testForSizeZero() {
+  assert(size(0) == '\0');
+}
+
+void testForSmall(){
+  assert(size(UPPER_BOUNDARY_FOR_SMALL -1) == 'S');
+  assert(size(UPPER_BOUNDARY_FOR_SMALL) == 'S');
+  assert(size(UPPER_BOUNDARY_FOR_SMALL+1) != 'S');

```

Comment:
what is this value expected to be? you may add a assert to check that !

Reply:
changes done.